rllib/algorithms/algorithm_config.py

```diff
@@ -997,19 +997,26 @@ def validate(self) -> None:
             else:
                 self.rl_module_spec = default_rl_module_spec
 
-            if (
-                self.exploration_config
-                and self._validate_exploration_conf_and_rl_modules
-            ):
-                # This is not compatible with RLModules, which have a method
-                # `forward_exploration` to specify custom exploration behavior.
-                raise ValueError(
-                    "When RLModule API are enabled, exploration_config can not be set."
-                    "If you want to implement custom exploration behaviour, "
-                    "please modify the `forward_exploration` method of the RLModule "
-                    "at hand. On configs that have a default exploration config, "
-                    "this must be done with `config._exploration_config={}`."
-                )
+            if self.exploration_config:
+                if self._validate_exploration_conf_and_rl_modules:
+                    # This is not compatible with RLModules, which have a method
+                    # `forward_exploration` to specify custom exploration behavior.
+                    raise ValueError(
+                        "When RLModule API are enabled, exploration_config can not be "
+                        "set. If you want to implement custom exploration behaviour, "
+                        "please modify the `forward_exploration` method of the "
+                        "RLModule at hand. On configs that have a default exploration "
+                        "config, this must be done with "
+                        "`config._exploration_config={}`."
```

Comment:
`config.exploration_config = {}` There should be no underscore.

Reply:
Thanks!